Python/pathconfig.c

```diff
@@ -265,7 +265,7 @@ config_init_module_search_paths(PyConfig *config, _PyPathConfig *pathconfig)
 
     const wchar_t *sys_path = pathconfig->module_search_path;
     const wchar_t delim = DELIM;
-    const wchar_t *p = sys_path;
+    const wchar_t *p;
```

Comment:
Would you mind to move the variable declaration inside the loop to even show better its scope?